modules/katamari/src/test/ts/atomic/api/arr/ArrFromTest.ts

```diff
@@ -6,8 +6,7 @@ import * as Arr from 'ephox/katamari/api/Arr';
 describe('atomic.katamari.api.arr.ArrFromTest', () => {
   it('works with arguments values', () => {
 
-    // eslint-disable-next-line prefer-arrow/prefer-arrow-functions,@typescript-eslint/no-unused-vars
```

Comment:
What happened to this rule is that removed? I think we should have a rule to enforce prefer-arrow-functions or is this smarter and detects the use of `arguments` inside it?

Reply:
I think the updated version is smarter, I looked up the docs and it talks about "equivalent conversions" (i.e. use of `arguments` or `this` mean it can't be an arrow)